website/content/docs/commands/pki/health-check.mdx

```diff
@@ -309,6 +414,23 @@ This health check verifies that auto-tidy is enabled, with sane defaults for int
 
 This health check verifies that tidy has run within the last run window. This can be critical/warning alerts as this can start to seriously impact Vault's performance.
 
+**Remediation Steps**:
+
+Read the tidy status endpoint, `vault read <mount>/tidy-status`, for possible error messages of why tidy has failed to run in the past, otherwise schedule a manual run of tidy or consider configuring auto-tidy
+
+To manually run a tidy operation:
```

Comment:
This doesn't match with the below fwiw.

Reply:
I'd still recommend auto-tidy here, but would perhaps still do a manual tidy.

Reply:
Fixed